packages/docusaurus-plugin-content-docs/src/index.js

```diff
@@ -220,38 +220,20 @@ class DocusaurusPluginContentDocs {
     });
   }
 
-  configureWebpack(config, isServer) {
+  configureWebpack(config, isServer, {getBabelLoader, getCacheLoader}) {
     const versionedDir = path.join(this.context.siteDir, 'versioned_docs');
     const translatedDir = path.join(this.context.siteDir, 'translated_docs');
 
     return {
       module: {
         rules: [
           {
-            test: /(\.mdx?)$/, // TODO: Read only this plugin's markdown files.
```

Comment:
We still need this TODO. Each plugin should read its own files only IMO. Otherwise the docs' markdown loader will also apply to the blog's and vice versa.

Reply:
The code below already do that.

```js
include: [this.contentPath],
```

This is webpack https://webpack.js.org/configuration/module/#ruleinclude
It's not so clear on the webpack documentation but if you try this out locally. You can see that the docs markdown loader only apply to docs. Blog markdown loader also only apply to blog.

Reply:
P.S: Let me try to verify it again and I'll try to update the PR

Reply:
Just tried this out again locally.
Note that we have mdx-loader on blog.
I tried to remove the include path on docs and it complains that there is no loader that can handle the file
![image](https://user-images.githubusercontent.com/17883920/55747036-e156ad80-5a6d-11e9-99eb-aae16333a8e2.png)

Another reference:

https://stackoverflow.com/questions/34623229/webpack-loaders-and-include
> include - defines the set of path or files where the imported files will be transformed by the loader.

Reply:
My bad. It's too far down I didn't see it. Good job!!!